tardis/util/base.py

```diff
@@ -32,7 +32,7 @@
 ATOMIC_SYMBOLS_DATA = (
     pd.read_csv(
         get_internal_data_path("atomic_symbols.dat"),
-        delim_whitespace=True,
+        sep=r"\s+",
```

Comment:
Could you add a comment to explain the regex?

Reply:
Done in commit: 945d86cee2f02d6fdff28a4a03e48af2fb9b0a25.